src/KlaviyoReactNativeSdk.ts

```diff
@@ -6,15 +6,8 @@ const LINKING_ERROR =
   '- You rebuilt the app after installing the package\n' +
   '- You are not using Expo Go\n';
 
-// @ts-expect-error
-const isTurboModuleEnabled = global.__turboModuleProxy != null;
-
-const KlaviyoReactNativeSdkModule = isTurboModuleEnabled
```

Comment:
This was supposed to be the basis of supporting NativeModules or TurboModules in the same lib. We'll probably have to revive this in the next phase so that we can maintain NM support when we add TM.